tests/apis/parameters.spec.ts

```diff
@@ -0,0 +1,139 @@
+        requireMock('@sap/plugin', sapPlugin);
+
+        const parameterValue = await getParameter(parameterName);
+        expect(parameterValue).to.be.undefined;
+        requireMock.stop('@sap/plugin');
+    });
+
+    it("@sap/plugin loaded, configuration is undefined ---> returns undefined", async () => {
+        stubEnv({ WS_BASE_URL: WS_BASE_URL });
+        const requireMock = require('mock-require');
+
+        const sapPlugin = {
+            window: {
+                configuration: () => undefined
+            }
+        };            
+        requireMock('@sap/plugin', sapPlugin);
+
+        const parameterValue = await getParameter(parameterName);
+        expect(parameterValue).to.be.undefined;
+        requireMock.stop('@sap/plugin');
+    });
+
+    it("@sap/plugin loaded, parameter name doesn't exist in the configuration ---> returns undefined", async () => {
+        stubEnv({ WS_BASE_URL: WS_BASE_URL });
+        const requireMock = require('mock-require');
+
+        const configuration = {};
+        const sapPlugin = {
+            window: {
+                configuration: () => configuration
+            }
+        };            
+        requireMock('@sap/plugin', sapPlugin);
+
+        const parameterValue = await getParameter(parameterName);
+        expect(parameterValue).to.be.undefined;
+        requireMock.stop('@sap/plugin');
+    });
+
+    it("@sap/plugin loaded, parameter name exists in the configuration ---> returns the parameter value", async () => {
+        stubEnv({ WS_BASE_URL: WS_BASE_URL });
+        const requireMock = require('mock-require');
+        const expectedParameterValue = "param1value";
+        const configuration = {param1: expectedParameterValue};
+        const sapPlugin = {
+            window: {
+                configuration: () => configuration
+            }
+        };            
+        requireMock('@sap/plugin', sapPlugin);
+
+        const parameterValue = await getParameter(parameterName);
+        expect(parameterValue).to.be.equal(expectedParameterValue);
+        requireMock.stop('@sap/plugin');
+    });
+
+    it("@sap/plugin loaded, parameter name exists in the configuration with null value ---> returns the parameter value", async () => {
+        stubEnv({ WS_BASE_URL: WS_BASE_URL });
+        const requireMock = require('mock-require');
+        const expectedParameterValue = null;
+        const configuration = {param1: expectedParameterValue};
+        const sapPlugin = {
+            window: {
+                configuration: () => configuration
+            }
+        };            
+        requireMock('@sap/plugin', sapPlugin);
+
+        const parameterValue = await getParameter(parameterName);
+        expect(parameterValue).to.be.equal(expectedParameterValue);
+        requireMock.stop('@sap/plugin');
+    });
+
+    it("@sap/plugin loaded, parameter name exists in the configuration with undefined value ---> returns the parameter value", async () => {

```

Comment:
like before you may not need to check all edge cases of parameters values (null/undefined) and instead can rely on member access syntax `["foo"]`